core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -274,12 +274,20 @@ protected void startJvmMonitorProcess() {
    * @param stopMessage empty string or the message that the master loses the leadership
    */
   protected void startServing(String startMessage, String stopMessage) {
+    startCommonServing(startMessage);
+    startLeaderServing(startMessage, stopMessage);
+  }
+
+  protected void startCommonServing(String startMessage) {
     MetricsSystem.startSinks(ServerConfiguration.get(PropertyKey.METRICS_CONF_FILE));
-    startServingRPCServer();
     LOG.info("Alluxio master web server version {} starting{}. webAddress={}",
         RuntimeConstants.VERSION, startMessage, mWebBindAddress);
     startServingWebServer();
     startJvmMonitorProcess();
+  }
+
+  protected void startLeaderServing(String startMessage, String stopMessage) {
```

Comment:
```suggestion
  protected void startRpcServer(String startMessage, String stopMessage) {
```

Reply:
Maybe rpc server can be common service in the future, this is why I named it to leaderServing to be more abstract.

Reply:
ok